### PR TITLE
fix: remove instruction hints and update readme

### PR DESCRIPTION
**Pull request by moT01**
Remove instruction hints and update readme.

Related to: https://github.com/freeCodeCamp/freeCodeCamp/issues/39720

Edit: We should probably wait on merging this until [the associated PR](https://github.com/freeCodeCamp/freeCodeCamp/pull/39954) hits production

### CODE REVIEW

**Comment by ShaunSHamilton** [comment]
Any particular reason for all of the empty lines in `server.js`?

**Comment by moT01** [comment]
It's where users will enter their code. Wasn't quite sure what to do there - I [added this in the instructions that would be on learn.](https://github.com/freeCodeCamp/freeCodeCamp/pull/39954/files#diff-1c2e9b709b5767f7522b4372cfe8c7c1R19) - Any thoughts?

File: README.md [hunk 1]
```diff
@@ -1,4 +1 @@
-**FreeCodeCamp** - bcrypt
-----
-
-[![Run on Repl.it](https://repl.it/badge/github/freeCodeCamp/boilerplate-bcrypt)](https://repl.it/github/freeCodeCamp/boilerplate-bcrypt)
\ No newline at end of file
+# [bcrypt Challenges](https://www.freecodecamp.org/learn/information-security/information-security-with-helmetjs/understand-bcrypt-hashes)
```
Comment (ShaunSHamilton) [inline README.md]:
```suggestion
# [Information Security with HelmetJS - BCrypt](https://www.freecodecamp.org/learn/information-security/information-security-with-helmetjs/understand-bcrypt-hashes)
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
I would prefer a heading like this

File: README.md (hunk 1 above)
Comment (moT01) [reply README.md]: me too 👍

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 

Thanks for the back-and-forth, Tom

**Review by RandellDawson — APPROVED** [review]
LGTM  We can merge this once PR https://github.com/freeCodeCamp/freeCodeCamp/pull/39954 hits production.

**Comment by ShaunSHamilton** [comment]
Thank you, for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Pull request merged**